allennlp/models/encoder_decoders/wikitables_semantic_parser.py

```diff
@@ -107,6 +120,12 @@ def __init__(self,
         self._action_padding_index = -1  # the padding value used by IndexField
         action_embedding_dim = nonterminal_embedder.get_output_dim() * 2
         num_entity_types = 2  # TODO(mattg): get this in a more principled way somehow?
+        self._embedding_dim = question_embedder.get_output_dim()
+        self._type_params = torch.nn.Linear(num_entity_types, self._embedding_dim)
+        self._neighbor_params = torch.nn.Linear(self._embedding_dim, self._embedding_dim)
+        # TODO(mattg): ok to leave number 1 since it gets squeezed out?
```

Comment:
I think it requires two parameters, I'll add a comment explaining the 1?

Reply:
Oh, right, sorry, you're using the `Linear` module, not just instantiating a parameter vector.  You're right, this is fine.  This will add a bias on top of just instantiating a parameter vector, which is actually probably what we want.